src/core/technologies/TechnologiesProvider.tsx

```diff
@@ -7,9 +7,11 @@ namespace TechnologiesProvider {
     loading: boolean;
     error: string;
     technologies: Technology[];
+    getTechnologies?(query?: string): void;
   }
 
   export interface Props {
+    getOnInit?: boolean;
```

Comment:
Same as above

Reply:
Also imo getOnInit should be replaced with getOnMount - refers to react mount cycle hook